origocli/commands/pipelines/instances.py

```diff
@@ -0,0 +1,195 @@
+import json
+import inquirer
+from inquirer.errors import ValidationError
+from fuzzywuzzy import process
+
+from origo.data.dataset import Dataset
+from origo.pipelines.resources.pipeline_instance import PipelineInstance
+
+from origocli.commands.pipelines.base import BasePipelinesCommand
+from origocli.output import create_output
+
+
+class PipelinesLsInstances(BasePipelinesCommand):
+    """
+    usage: origo pipelines ls-instances --pipeline-arn=<pipeline-arn> [options]
```

Comment:
Why a separate command to list instances by pipeline, instead of combining with `PipelineInstanceLs`?

Reply:
The command is not included in the usage section of `PipelineInstances` below.

Reply:
Just transferring from existing functionality for now, not removing or restructuring commands. I'll add a ticket to look into it